packages/dx-react-chart-demos/src/demo-sources/layout/material-ui/legend.jsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import Paper from '@material-ui/core/Paper';
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  BarSeries,
+  Legend,
+  Title,
+} from '@devexpress/dx-react-chart-material-ui';
+import { withStyles } from '@material-ui/core/styles';
+
+import { Stack } from '@devexpress/dx-react-chart';
+
+import { ageStructure } from '../../../demo-data/data-vizualization';
+
+const styles = {
+  root: {
+    width: '60%',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+  },
+  title: {
+    margin: '0px',
+  },
+  legendRoot: {
+    display: 'flex',
+    flexDirection: 'row',
+  },
+  item: {
+    flexDirection: 'column',
+    margin: 'auto',
+  },
+  label: {
+    margin: '10px',
+  },
+};
+
+const RootWithTitle = withStyles(styles)(({ classes, ...restProps }) => (
+  <div className={classes.root}>
+    <Title.Text className={classes.title} text="👪 Population" />
```

Comment:
Legend title seems too large for a legend.
Since there is already no common code for `RootWithTitle` (jsxt) more specific html elements can be used (*h* elements or something like that in material).

Reply:
ok